sstables/exceptions.hh

```diff
@@ -13,10 +13,20 @@
 #include <seastar/core/format.hh>
 
 #include "sstables/component_type.hh"
+#include "sstables/stats.hh"
 #include "seastarx.hh"
 
 namespace sstables {
+
 class malformed_sstable_exception : public std::exception {
+    class malformed_accounter {
+        sstables_stats _stats;
+    public:
+        malformed_accounter() {
+           _stats.on_malformed_sstable_exception();
+        }
+    };
+    malformed_accounter _accounter;
```

Comment:
This member seems unnecessary, why not just bump this stat in the `malformed_sstable_exception` constructor directly?

Reply:
there were two constructors one calling the other, and I thought this would be nice to prevent accidental double bumps.